### PR TITLE
Add Butil to boilerplate (#6379)

### LINKED ISSUE #6379: Add `Buitl` to the `Boilerplate` project template
Add `Buitl` to the `Boilerplate` project template

### PR DESCRIPTION
**Pull request by msynk**
closes #6379

### CODE REVIEW

File: src/Templates/Boilerplate/Bit.Boilerplate/src/Client/Boilerplate.Client.Core/Components/Layout/Footer.razor.cs [hunk 1]
```diff
@@ -23,7 +24,13 @@ protected override Task OnInitAsync()
 
     private async Task OnCultureChanged()
     {
-        await JSRuntime.SetCookie(".AspNetCore.Culture", $"c={SelectedCulture}|uic={SelectedCulture}", expiresIn: 30 * 24 * 3600, rememberMe: true);
+        await cookie.Set(new ButilCookie
```
Comment (ysmoradi) [inline src/Templates/Boilerplate/Bit.Boilerplate/src/Client/Boilerplate.Client.Core/Components/Layout/Footer.razor.cs:27]: Use
```csharp
await cookie.Set(new()
...
```
instead

File: src/Templates/Boilerplate/Bit.Boilerplate/src/Client/Boilerplate.Client.Core/Services/AuthenticationManager.cs [hunk 2]
```diff
@@ -92,7 +92,12 @@ private async Task StoreToken(TokenResponseDto tokenResponseDto, bool? rememberM
         await storageService.SetItem("refresh_token", tokenResponseDto!.RefreshToken, rememberMe is true);
         if (AppRenderMode.PrerenderEnabled && AppRenderMode.IsBlazorHybrid is false)
         {
-            await jsRuntime.SetCookie("access_token", tokenResponseDto.AccessToken!, tokenResponseDto.ExpiresIn, rememberMe is true);
+            await cookie.Set(new ButilCookie
```
Comment (ysmoradi) [inline src/Templates/Boilerplate/Bit.Boilerplate/src/Client/Boilerplate.Client.Core/Services/AuthenticationManager.cs:95]: Use
```csharp
await cookie.Set(new()
...
```
instead.

**Review by ysmoradi — CHANGES_REQUESTED** [review]
REJECT!

File: src/Templates/Boilerplate/Bit.Boilerplate/src/Client/Boilerplate.Client.Core/Components/Layout/Footer.razor.cs (hunk 1 above)
Comment (msynk) [reply src/Templates/Boilerplate/Bit.Boilerplate/src/Client/Boilerplate.Client.Core/Components/Layout/Footer.razor.cs:27]: yeah you would think, but no, it's not working like this :)

File: src/Templates/Boilerplate/Bit.Boilerplate/src/Client/Boilerplate.Client.Core/Services/AuthenticationManager.cs (hunk 2 above)
Comment (msynk) [reply src/Templates/Boilerplate/Bit.Boilerplate/src/Client/Boilerplate.Client.Core/Services/AuthenticationManager.cs:95]: same

**Comment by msynk** [comment]
YOUR REVIEW IS REJECTED!

**Review by ysmoradi — DISMISSED** [review]

**Pull request merged**